pages/common/vim.md

```diff
@@ -1,6 +1,6 @@
 # vim
 
-> Vi IMproved, a programmer's text editor.
+> Vi IMproved, a programmers text editor.
```

Comment:
I think this change goes the wrong way. You'd say "a woman's best friend", not "a women best friend"; or "a scientist's tool", not "a scientists tool" (sorry for the silly examples, it's what came to me from the top of my head)